app/controllers/application_controller.rb

```diff
@@ -1,3 +1,12 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  before_action :get_favorite#, only: [:new, :create]
+
+  private
+
+   def get_favorite
+     @favorite = Favorite.new(session[:favorite])
+   end
```

Comment:
have a session active on every page, every time we navigate to a new page

Reply:
Look at the session before the lesson!